server/utils/assessmentUtils.test.ts

```diff
@@ -417,4 +428,28 @@ describe('assessmentUtils', () => {
       expect(informationSetAsNotReceived(assessment)).toEqual(false)
     })
   })
+
+  describe('getSections', () => {
+    const assessment = assessmentFactory.build({ status: 'pending' })
+
+    it('returns all sections if informationSetAsNotReceived is false', () => {
+      assessment.data = { 'sufficient-information': { 'information-received': { informationReceived: 'yes' } } }
+
+      const sections = getSections(assessment)
+      const sectionNames = sections.map(s => s.name)
+
+      expect(sections.length).toEqual(Assess.sections.length)
+      expect(sectionNames).toContain('AssessApplication')
+    })
+
+    it('returns all sections if informationSetAsNotReceived is true', () => {
```

Comment:
Is this description correct? We're checking that it *doesn't* return all sections, right?

And to match the description, should these two tests be mocking `informationSetAsNotReceived` rather than setting the assessment data?

Reply:
Yeah, good call. To avoid having to mock the same module, I've changed the description too, and added a test for when the answer has not been completed.